From 510dd87deeca370946a7b9d8e440768eb71141e2 Mon Sep 17 00:00:00 2001
From: Yi Kong <yikong@google.com>
Date: Tue, 25 Jun 2019 23:21:57 +0800
Subject: [PATCH 1/8] Opt libpdfium out of the new pass manager

The new pass manager is causing non-deterministic build.

Test: TreeHugger
Bug: 135660678
Change-Id: Ib4407e7f9bb11138e868d81ccafc63110ec88d45
---
 Android.bp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Android.bp b/Android.bp
index ab7a5981b..550d9341f 100644
--- a/Android.bp
+++ b/Android.bp
@@ -18,6 +18,10 @@ cc_defaults {
         // and cpwl_edit_impl.cpp:1825 have -Wimplicit-fallthrough.
         "-Werror",
 
+        // The new pass manager causes non-deterministic build result.
+        // http://b/135660678
+        "-fno-experimental-new-pass-manager",
+
         "-DOPJ_STATIC",
         "-DPNG_PREFIX",
         "-DPNG_USE_READ_MACROS",
-- 
2.40.1

